dockerfiles/centos/centos-7/centos-7-base/Dockerfile

```diff
@@ -56,6 +56,9 @@ RUN yum -y install epel-release && \
     mkdir  /etc/vncskel/.vnc && \
     echo "" | vncpasswd -f > /etc/vncskel/.vnc/passwd && \
     chmod 0600 /etc/vncskel/.vnc/passwd && \
+    # Repo is a useful tool for checking out multiple git repos
+    wget https://storage.googleapis.com/git-repo-downloads/repo -O /usr/local/bin/repo && \
```

Comment:
Since this isn't coming from the package manager and thus doesn't do verification automatically, it should at least do a checksum comparison against an expected version.

Reply:
@rewitt1 @moto-timo: repo is a tool I want to be able to use in crops containers so taking a look at this. The issue is that the URL https://storage.googleapis.com/git-repo-downloads/repo gets you the latest release of repo and so we can't expect it to match any given checksum for long. repo really doesn't support running anything other than the very latest commit, it actually pulls the latest commit from its own git repository when you run it. Sadly it's a widely used tool and we do need to try to support it.

The https cert for storage.goolgeapis.com will be checked as part of this download so that should be safe.

Reply:
The source of most of this PR was from #22 including the Intel copyright (I don't think I did much modification of any of his work). I think #22 was just copy paste of existing scripts. We are currently installing repotool into the running container, so I have more or less stopped working on this. It is a frequent enough request to warrant further work. It would be great if you can pick this up @pbkr